code/modules/clothing/head/jobs.dm

```diff
@@ -146,19 +146,51 @@
 	dog_fashion = /datum/dog_fashion/head/warden
 
 /obj/item/clothing/head/warden/drill
-	name = "warden's drill hat"
-	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection. Has the letters 'FMJ' enscribed on its side."
+	name = "warden's campaign hat"
+	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
 	icon_state = "wardendrill"
 	item_state = "wardendrill"
 	dog_fashion = null
+	var/anger = 1

```

Comment:
I would prefer defines instead of magic numbers for modes.

Reply:
well i looked at hailer code so i based it off of that mostly

Reply:
cancer

Reply:
I still want defines used